README.md

```diff
@@ -1 +1,225 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+ 
+  <br/>
+
+  <h3><b>"Hello Microverse"</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#A-program-for-greeting-users)
+  - [🛠 Built With](#built-with-HMTL-CSS)
+    - [Tech Stack](#HTML-CSS)
+    - [Key Features](#Header)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello-Microverse] <a name="A-program-for-greeting-users"></a>
+
+
+
+**[Hello-Microverse]** is a program intended to greet users.
+
+## 🛠 Built With <a name="HTML-CSS"></a>
+
+### Tech Stack <a name="HTML-CSS"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
```

Comment:
As requested by the previous reviewer, please kindly update the tech stack to the scope of this project by removing **React, Express, and Postgresql** and including **HTML AND CSS** in the Client dropdown. 

👇 
```
<details>
  <summary>Client</summary>
  <ul>
    <li><a href="https://reactjs.org/">HTML</a></li>
    <li><a href="https://reactjs.org/">CSS</a></li>
  </ul>
</details>
```